please.sh

```diff
@@ -777,7 +788,7 @@ build_and_test_64 () {
 		fi &&
 		if '"$(if test -z "$skip_tests"
 			then
-				echo "! ${make_t_prefix}make -C t -j5 -k"
+				echo "! ${make_t_prefix}make -C t -j5 -k | ${clean_t}"
```

Comment:
The curly braces are only necessary in `${make_t_prefix}` to separate it from `make` (which would otherwise be interpreted as part of the variable name). For `clean_t`, it is not necessary, so let's not do that.